meson.build

```diff
@@ -1,12 +1,15 @@
-project('com.github.danrabbit.harvey', ['vala', 'c'],
+project(
+    'com.github.danrabbit.harvey',
+    ['vala', 'c'],
```

Comment:
This should be in a different PR. These code style changes are not related to i18n changes